ctypesgen/printer_python/printer.py

```diff
@@ -133,9 +133,7 @@ def print_preamble(self):
         self.file.write("# Begin preamble for Python\n\n")
         if self.options.embed_preamble:
             with open(PREAMBLE_PATH, "r") as preamble_file:
-                preamble_file_content = preamble_file.read()
-                filecontent = preamble_file_content.replace("# ~POINTER~", "")
-                self.file.write(filecontent)
+                self.file.write(preamble_file.read())
```

Comment:
this is done more nicely in the new codebase via an `_embed_file()` helper that uses a paragraph context and `shutil.copyfileobj()`.